src/lang/string_view_utf8.hpp

```diff
@@ -221,4 +221,27 @@ class string_view_utf8 {
         s.type = EType::NULLSTR;
         return s;
     }
+
+    /// string view has same resource
+    constexpr bool operator==(const string_view_utf8 &other) const {
+        if (type != other.type)
+            return false; // type missmatch
+
+        switch (type) {
+        case EType::RAM:
+        case EType::CPUFLASH:
+            return attrs.cpuflash.utf8raw == other.attrs.cpuflash.utf8raw;
+        case EType::FILE:
+            return (attrs.file.f == other.attrs.file.f) && (attrs.file.startOfs == other.attrs.file.startOfs);
+        case EType::SPIFLASH:
```

Comment:
SPIFLASH and USBFLASH types are equal by default as well? That's not true

Reply:
added assert